docs/quick_tutorial/static_assets.rst

```diff
@@ -43,13 +43,18 @@ Steps
    .. literalinclude:: static_assets/tutorial/static/app.css
     :language: css
 
-#. Make sure we haven't broken any existing code by running the tests:
+#. Add a functional test that requires the newly added static file to be delivered:
+
+   .. literalinclude:: static_assets/tutorial/tests.py
+    :linenos:
```

Comment:
Quick Tutorial has been neglected. In other tutorials, we only recently started to implement `:lineno-match:` and `:pyobject:` (and other options) to replace the somewhat lazy `:linenos:` along with the more difficult to maintain and problematic `:lines:` and `:lineno-start:`. This would be a start for Quick Tutorial.

In this case, the line numbers tell the user where to place the snippet, which they can still copy-pasta.

Also the source files are available in the repository, If the user wants to just drag and drop or copy-pasta from them. I just realized that we don't advertise that in this tutorial, but we do in others, which is an oversight. I've created a new issue #3154 to track that. If you're feeling ambitious, you're welcome to it!